File: ezbiz-backend/internal/model/social.go
package model

type Social struct {
	Id          int64  `json:"id" db:"id"`
	Name        string `json:"name" db:"name"`
	Url         string `json:"url" db:"url"`
	ImagePath   string `json:"imagePath" db:"image_path"`
	Placeholder string `json:"placeholder" db:"placeholder"`
	CreatedAt   string `json:"createdAt" db:"created_at"`
	UpdatedAt   string `json:"updatedAt" db:"updated_at"`
}

File: ezbiz-admin/src/interfaces/social.ts
import { z } from "zod"

export const SocialSchema = z.object({
  id: z.number().optional(),
  name: z.string().min(2, { message: "Name is too short" }),
  url: z.string().optional(),
  imagePath: z.string().optional(),
  placeholder: z.string().optional(),
  createdAt: z.string().optional(),
  updatedAt: z.string().optional(),
})

export type SocialValues = z.infer<typeof SocialSchema>
